src/utils.ts

```diff
@@ -43,10 +42,21 @@ export const checkIfNodeOrFileProtocol = (modulePath: string) => {
   return isNode || isFile;
 };
 
-export const resolveModulePath = (specifier: string, cacheMapPath: string) => {
-  const modulePath = importmap.resolve(specifier, cacheMapPath);
-  log.debug("resolveModulePath:", { modulePath });
-  return modulePath;
+export const resolveModulePath = (specifier: string, cacheMapPath: string): string | null => {
+  try {
+    const modulePath = importmap.resolve(specifier, cacheMapPath);
+    log.debug("resolveModulePath:", { modulePath });
+    return modulePath;
+  } catch {
+    /*
+      If a module is not found in the importmap, we should just let the loader to skip it.
+      - Users might have installed it using `npm`.
+      - It can be a node-js dependency. (e.g. `assert`). We detect builtins using `node:` protocol.
+        But it is still not widely adopted yet.
+    */
+    log.debug(`Failed in resolving ${specifier} in ${cacheMapPath}`);
+    return null;
```

Comment:
What 'value" does null provide here?

Reply:
This line here know, that the specifier is failed to resolve from the map and can be skipped.
https://github.com/jspm/node-importmap-http-loader/blob/skip-missing-dependency-from-importmap/src/loader.ts#L39

Reply:
I mean that I don't see a reason why returning null is better than nothing. 😬 Sorry I wasn't clearer! 😃